<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QDateEdit" name="birthday_edit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>E-Mail:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="email_edit"/>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Phone:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="phone_edit"/>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="QTextEdit" name="description_edit"/>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Birthday:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="username_edit"/>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nameï¼š</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="address_edit"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
